### PR TITLE
Better ShellCheck installer

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun @ricmatsui 

This adds better output to the ShellCheck installer

### CODE REVIEW

File: bin/circleci-install-shellcheck [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+
 if [ ! -f $BIN_PATH ]; then
+  echo "No ShellCheck found"
   install_shellcheck
 else
-  CURRENT=$($BIN_PATH -V | grep version: | cut -d: -f2 | awk '{print $1}')
-  if [ "$VERSION" != "$CURRENT" ]; then
+  EXISTING_VERSION=$(currently_installed_shell_check_version)
+  if [ "$VERSION" != "$EXISTING_VERSION" ]; then
+    echo "An older version of ShellCheck ($EXISTING_VERSION) was found"
```
Comment (allewun) [inline bin/circleci-install-shellcheck]: Could even be a newer version? Change text accordingly

File: bin/circleci-install-shellcheck [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
+
+# Installs the current version of ShellCheck
+function install_shellcheck {
+  echo "Installing ShellCheck $VERSION"
   cabal update --verbose=0
   cabal install --verbose=0 shellcheck-$VERSION
 }
 
+
+function currently_installed_shell_check_version {
```
Comment (allewun) [inline bin/circleci-install-shellcheck]: nitpicky: `shell_check` -> `shellcheck`

**Comment by allewun** [comment]
LGTM

File: bin/circleci-install-shellcheck (hunk 2 above)
Comment (kevgo) [inline bin/circleci-install-shellcheck]: Good catch. Fixed

File: bin/circleci-install-shellcheck (hunk 1 above)
Comment (kevgo) [inline bin/circleci-install-shellcheck]: Fixed

**Pull request closed without merging**